recipes/glib/all/conanfile.py

```diff
@@ -81,7 +81,7 @@ def requirements(self):
             self.requires("libiconv/1.17")
 
     def build_requirements(self):
-        self.tool_requires("meson/1.2.2")
+        self.tool_requires("meson/[>=1.2.3]")
```

Comment:
```suggestion
        self.tool_requires("meson/[>=1.2.3 <2]")
```
if we are doing this it should be capped to the next major version :)